docs/source/ros_api.rst

```diff
@@ -85,6 +66,8 @@ odom (`nav_msgs/Odometry`_)
   enabled in the :doc:`configuration`, this topic will be used as input for
   SLAM.
 
+.. TODO: add NavSatFix? Landmarks?
```

Comment:
Feel free to document them. NavSatFix is unavailable in 2D and optional in 3D, its lua parameter is use_nav_sat

Reply:
Not for now, maybe I'll do landmarks at some point.